### ISSUE TITLE
Eye of Ender cannot be thrown

### ISSUE THREAD

**Issue by mushroomhostage** [mushroomhostage/exphc#58]
Unable to throw the Eyes of Ender. Right-clicking, nothing happens.

**Comment by mushroomhostage** [comment]
Maybe a mod conflict?
http://www.minecraftwiki.net/wiki/Data_values#Entity_IDs entity ID 15 = Thrown Eye of Ender

Mo' Creatures spawnitem/15 spawns a fox.. but SilkSpawners shows:

09:22:43 [INFO] Entity 15 = EyeOfEnderSignal

**Comment by mushroomhostage** [comment]
Removing Mo' Creatures and RanCraft Penguins, from client and server, no different (still can't throw ender eyes). Same with Elemental Creepers..

Also tried removing all plugins, no change.

**Comment by mushroomhostage** [comment]
Reproduces even with an empty "mods" folder. Does not reproduce in single player. Maybe a jar mod problem?

Reproduces with craftbukkit-1.2.5-R1.3-MCPC-SNAPSHOT-91+trees3+sm8.6+4096v4+ds+thx.jar.

Does not reproduce with clean MCPC 91 + THX v018 r2. 
_Does_ reproduce with clean MCPC 91 + Trees++ 1.3.35..that's odd (not with Trees++ on vanilla server)

**Comment by mushroomhostage** [comment]
Trees++ does change WorldGenStronghold, but only to add its list of biomes. 

Added debugging to ItemEnderEye, a() 

```
        ChunkPosition chunkposition = world.b("Stronghold", (int)entityhuman.locX, (int)entityhuman.locY, (int)entityhuman.locZ);
        System.out.println("Stronghold location ="+chunkposition);
```

stronghold location is null! World's public ChunkPosition b(String s, int i, int j, int k) calls q().findNearestMapFeature(this, s, i, j, k);.  in IChunkProvider. ChunkProviderGenerate implements, if("Stronghold".equals(s1) && u != null) then calls u.getNearestGeneratedFeature(world, i1, j1, k1); where u = WorldGenStronghold. getNearestGeneratedFeature inherited from StructureGenerator.

**Comment by mushroomhostage** [comment]
Problem was in WorldGenStronghold base edit, left decompiled func_40203_a() as is, when it should've been changed to a(). Fixed in LCTrees++ 1.3.35 r4 port: http://www.mcportcentral.co.za/index.php?topic=2224.0 - deployed on ExpHC, can now throw Eyes of Ender.
